src/utils/gz.js

```diff
@@ -1,19 +1,33 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
 
 import Worker from './worker-factory';
 
 const zeeWorker = new Worker('zee-worker');
 const zeeCallbacks = [];
 
-zeeWorker.onmessage = function(msg) {
-  zeeCallbacks[msg.data.callbackID][msg.data.type](msg.data.data);
-  zeeCallbacks[msg.data.callbackID] = null;
+type ZeeWorkerData = {
+  callbackID: number,
+  type: 'success' | 'error',
+  data: any,
+};
+
+zeeWorker.onmessage = function(msg: MessageEvent) {
+  const data = ((msg.data: any): ZeeWorkerData);
```

Comment:
If Flow accepts it, a way to do the same thing without using `any` could be to type `msg` like this:
```javascript
zeeWorker.onmessage = function(msg: { data: ZeeWorkerData } & MessageEvent) { 
```